test/algorithm/test_view_cone.py

```diff
@@ -0,0 +1,112 @@
+""" test_view_cone.py """
+import unittest
+from random import randint
+
+from ddt import ddt, data
+
+from kaos.algorithm import view_cone, coord_conversion, TimeInterval, ViewConeFailure
+
+# Time constants for test environment:
+ONE_DAY = 86164 #23:56:04 in seconds
+J2000 = 946684800 # Jan 1st 1970 in POSIX
```

Comment:
We should put all the constants in one place. But that should probably be done with the refactoring. So I'm just making this comment to whine about not already having it in one place and also as a reminder to myself.

Reply:
Added a TODO

Reply:
Good idea, thanks!